docs/microbit_micropython_api.rst

```diff
@@ -266,6 +328,25 @@ There is an I2C bus on the micro:bit that is exposed via the `i2c` object.  It h
     # write buf to device with addr; repeat=True means a stop bit won't be sent.
     i2c.write(addr, buf, repeat=False)
 
+Sounds **V2**
+------
+
+A set of expressive sounds are available to the micro:bit **V2** and can be
+accessed via the ``audio`` module.
```

Comment:
They can be accessed via the `microbit` module and played with the `audio module`.

Reply:
UPdated